muesli/web/viewsLecture.py

```diff
@@ -89,25 +91,31 @@ def __call__(self):
                 'times': times,
                 'prefs': utils.preferences}
 
-@view_config(route_name='lecture_add_exam', renderer='muesli.web:templates/lecture/add_exam.pt', context=LectureContext, permission='edit')
+@view_config(route_name='lecture_add_exam', renderer='muesli.web:templates/lecture/add_exam.pt', context=AddExamContext, permission='edit')
```

Comment:
please remove AddExamContext here as well

Reply:
The new context is used to get the german translation for the category, which is needed, now that the implementation of how exams are created has changed.